boxsdk/client/client.py

```diff
@@ -143,7 +138,12 @@ def users(self, limit=None, offset=0, filter_term=None):
             params['filter_term'] = filter_term
         box_response = self._session.get(url, params=params)
         response = box_response.json()
-        return [User(self._session, item['id'], item) for item in response['entries']]
+        User = Translator().translate('user')
```

Comment:
Apparently pylint doesn't like uppercase names as variable names -_-

You can either pylint:ignore that, or change it to something like `user_class`.